### DISCUSSION TITLE
Updating a base image version should update also some variables

### DISCUSSION THREAD

**Discussion by orlandoalexandrescu** [renovatebot/renovate#17914] [Suggest an Idea]
### What would you like Renovate to be able to do?

Renovate should automate the following process: 

We have a docker image:

`FROM docker-upstream.somenetworkhost.net/postgres:11.17-alpine3.16 as upstream`

This was just automatically updated by Renovate from

`FROM docker-upstream.somenetworkhost.net/postgres:11.14-alpine3.15 as upstream`

In the Dockerfile, we also have a variable, called PG_VERSION, which used to be:

`PG_VERSION=11.14`

and should be automatically upgraded to 

`PG_VERSION=11.17`

### If you have any ideas on how this should be implemented, please tell us here.

Some sort of comment, similar to what is done here for Alpine apk packages handled with Repology as datasource, might help:

```
# renovate: datasource=repology depName=alpine_3_16/python3-dev versioning=loose
ARG PYTHON3_DEV_VERSION="3.10.5-r0"
```

### Is this a feature you are interested in implementing yourself?

No

**Comment by rarkins** [comment]
Everything you describe looks like it should be possible using the existing regex manager and package rules/grouping capabilities

**Comment by Churro** [comment]
What renovate version are you using? Support for updating ARG values in Dockerfiles was added with PR https://github.com/renovatebot/renovate/pull/15751

It won't work with version strings split across multiple sources though. A version definition like `11.17-alpine3.16` needs to be stored entirely within a variable.
